business_requirement_deliverable/models/business.py

```diff
@@ -189,14 +189,13 @@ def _compute_get_currency(self):
     @api.multi
     def _get_pricelist(self):
         for brd in self:
-            partner_id = False
-            if brd.business_requirement_id and (
-                brd.business_requirement_id.partner_id
-            ):
-                partner_id = brd.business_requirement_id.partner_id
-            if partner_id and partner_id.property_product_pricelist:
-                return partner_id.property_product_pricelist
-            return partner_id
+            if brd.business_requirement_partner_id:
+                partner_id = brd.business_requirement_partner_id
+                return (
+                    partner_id.property_product_estimation_pricelist or
+                    partner_id.property_product_pricelist)
+            else:
```

Comment:
@victormartinelicocorp I would remove the `else` since it's unnecessary. But more importantly, are you sure it's at the right level? It's an `@api.multi` and you return `False` at the first item, isn't strange to not finish the loop?